mono/metadata/loader.c

```diff
@@ -2629,7 +2630,8 @@ mono_method_get_header_checked (MonoMethod *method, MonoError *error)
 	gpointer loc;
 	MonoGenericContainer *container;
 
-	error_init (error);
+	MONO_API_ERROR_INIT (error);
```

Comment:
Why the double init?

Reply:
It is a form of commenting. The initialization is needed here for two reasons, so two inits. The init is cheap and maybe optimized away.

Reply:
I split the function into public _checked and internal _internal, so that _internal callers are required to init. I didn't update the callers yet, that can be separate. Consider that for the other four MONO_API_ERROR_INIT also.

Reply:
I moved the split into a separate PR.